ginga/gw/GingaGw.py

```diff
@@ -806,4 +815,11 @@ def page_closed_cb(self, widget, child, wsname):
             self.gui_delete_channel(channel.name)
 
 
+    ########################################################
+    ### NON-PEP8 PREDECESSORS: TO BE DEPRECATED
+
+    getFont = get_font
```

Comment:
Maybe use [functions in astropy.utils](http://docs.astropy.org/en/stable/utils/index.html#id3) like `deprecated` or `deprecated_attribute` (where appropriate) throughout Ginga, so it is clear that when the old-style is used, it should be updated?